Translations/pt_BR.po

```diff
@@ -9412,526 +9342,376 @@ msgid "Blood Stone"
 msgstr "Pedra Sanguínea"
 
 #: Source/translation_dummy.cpp:71
-#, fuzzy
-#| msgid "Hell Stone"
 msgctxt "monster"
 msgid "Hell Stone"
 msgstr "Pedra do Inferno"
 
 #: Source/translation_dummy.cpp:72
-#, fuzzy
-#| msgid "Lava Lord"
 msgctxt "monster"
 msgid "Lava Lord"
-msgstr "Senhor da Lava"
+msgstr "Lorde da Lava"
 
 #: Source/translation_dummy.cpp:73
-#, fuzzy
-#| msgid "Horned Demon"
 msgctxt "monster"
 msgid "Horned Demon"
 msgstr "Demônio com Chifres"
 
 #: Source/translation_dummy.cpp:74
-#, fuzzy
-#| msgid "Mud Runner"
 msgctxt "monster"
 msgid "Mud Runner"
 msgstr "Trilhador da Lama"
 
 #: Source/translation_dummy.cpp:75
-#, fuzzy
-#| msgid "Frost Charger"
 msgctxt "monster"
 msgid "Frost Charger"
 msgstr "Suicida Gélido"
 
 #: Source/translation_dummy.cpp:76
-#, fuzzy
-#| msgid "Obsidian Lord"
 msgctxt "monster"
 msgid "Obsidian Lord"
-msgstr "Senhor da Obsidiana"
+msgstr "Lorde da Obsidiana"
 
 #: Source/translation_dummy.cpp:77
-#, fuzzy
-#| msgid "oldboned"
 msgctxt "monster"
 msgid "oldboned"
 msgstr "Ossovelho"
 
 #: Source/translation_dummy.cpp:78
-#, fuzzy
-#| msgid "Red Death"
 msgctxt "monster"
 msgid "Red Death"
-msgstr "Morte Vermelha"
+msgstr "Morte Rubra"
```

Comment:
Slightly questionable this one. It makes it a bit more formal which I guess is fine.

Reply:
Yeah, maybe I went slightly too far here. After reading all the monsters' names, 'Rubra' sounds more like the rest of them.

I am fine with rolling back though.

Reply:
It should be fine. It is after all similar to how we use "Negro(a)" instead of "Preto(a)" for "black" when used as an adjective. The only difference is that "Rubro(a)" is not as commonly used, but it is still obviously correct.